src/Aether/Devices/Sensors/ObservableSgp4x.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Aether.CustomUnits;
+using Aether.Devices.Sensors.Metadata;
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnitsNet;
+
+namespace Aether.Devices.Sensors
+{
+    internal class ObservableSgp4x : ObservableSensor, IObservableI2cSensorFactory
+    {
+        private readonly Drivers.Sgp4x _sensor;
+        private static readonly SensorDependency TemperatureDependency = new SensorDependency(Measure.Temperature, true);
+        private static readonly SensorDependency HumidityDependency = new SensorDependency(Measure.Humidity, true);
+        private ObservableSgp4x(I2cDevice device)
+        {
+            _sensor = new Drivers.Sgp4x(device);
+            Start();
+        }
+
+        public static int DefaultAddress => Drivers.Sgp4x.DefaultI2cAddress;
+
+        public static string Manufacturer => "Sensirion";
+
+        public static string Name => "SGP4x";
+
+        public static string Uri => "https://www.sensirion.com/en/environmental-sensors/gas-sensors/sgp40/";
+
+        public static IEnumerable<MeasureInfo> Measures { get; } = new[]
+        {
+            new MeasureInfo(Measure.VOC)
+        };
+
+        public static IEnumerable<SensorDependency> Dependencies => new[] { TemperatureDependency, HumidityDependency };
+
+        public static IEnumerable<SensorCommand> Commands => SensorCommand.NoCommands;
+
+        public static ObservableSensor OpenSensor(I2cDevice device, IObservable<Measurement> dependencies) => new ObservableSgp4x(device);
+
+        protected override void DisposeCore() => _sensor.Dispose();
+
+        protected override async Task ProcessLoopAsync(CancellationToken cancellationToken)
```

Comment:
This needs to wire up dependencies still. See:

https://github.com/scalablecory/aether/blob/4c4d8dd47f6ca4c0c508cd80c11371c5f930dda1/src/Aether/Devices/Sensors/ObservableScd4x.cs#L29-L33

https://github.com/scalablecory/aether/blob/4c4d8dd47f6ca4c0c508cd80c11371c5f930dda1/src/Aether/Devices/Sensors/ObservableScd4x.cs#L52-L55